src/ray/core_worker/task_manager.cc

```diff
@@ -1169,6 +1169,21 @@ bool TaskManager::RetryTaskIfPossible(const TaskID &task_id,
                     rpc::TaskStatus::FAILED,
                     worker::TaskStatusEvent::TaskStateUpdate(error_info));
       task_entry.MarkRetry();
+      // Push the error to the driver if the task will still retry.
```

Comment:
Can we put this behind a feature flag? Since this goes through GCS and I'm a bit concerned that during excessive node preemption, we will tons of these messages and overload the GCS and driver.

Reply:
makes sense, sounds good

Reply:
done